app/static/lib/conp-react/src/charts/TotalDatasetsPipelines/index.js

```diff
@@ -152,35 +154,30 @@ const TotalDatasetsPipelines = (props) => {
         };
 
         chartData.datasets.elements.forEach(element => {
-            const dateAdded = new Date(element.dateAdded);
+            const dateAdded = new Date(element.dateAdded.concat('T00:00:00'));
```

Comment:
Because the format of the `dateAdded` field of the JSON response is of format `YYY-MM-DD`, the date is parsed in UTC (GMT) which resulted in returning a date for the day before.

See: https://stackoverflow.com/questions/7556591/is-the-javascript-date-object-always-one-day-off

Adding the `T00:00:00` will ensure that the date object created correspond to the correct day.